src/content/es/2019/markup.md

```diff
@@ -0,0 +1,261 @@
+---
+part_number: I
+chapter_number: 3
+title: Markup
+description: Capítulo sobre marcado del Web Almanac de 2019 que cubre elementos utilizados, elementos personalizados, valor, productos y casos de uso comunes.
+authors: [bkardell]
+reviewers: [zcorpan, tomhodgins, matthewp]
+discuss: 1758
+published: 2019-11-04T12:00:00+00:00:00
+last_updated: 2019-11-07T21:46:11.000Z 
+---
+
+## Introduction
+
+En 2005, Ian "Hixie" Hickson publicó [algunos análisis de datos de marcado](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html) basándose en varios trabajos anteriores. Gran parte de este trabajo tenía como objetivo investigar los nombres de las clases para ver si había una semántica informal común que los desarrolladores estaban adoptando y que podría tener sentido estandarizar. Parte de esta investigación ayudó a darle forma a nuevos elementos en HTML5.
+
+14 años después, es hora de dar un nuevo vistazo. Desde entonces, hemos tenido la introducción de [Elementos Personalizados](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) y del [Manifesto Web Extensible](https://extensiblewebmanifesto.org/) alentando a que encontremos mejores formas de pavimentar los caminos de acceso al permitir a los desarrolladores explorar el espacio de los elementos y permitir a los organismos de estándares [actuar más como editores de diccionario](https://bkardell.com/blog/Dropping-The-F-Bomb-On-Standards.html).  A diferencia de los nombres de clase CSS, que podrían usarse para cualquier cosa, podemos estar mucho más seguros de que los autores que usaron un *elemento* no estándar realmente pretendían que fuera un elemento.  
```

Comment:
Since MDN's content is partially translated to Spanish we could as well link to the non-localized MDN website and let them redirect to user's preferred version.
```suggestion
14 años después, es hora de dar un nuevo vistazo. Desde entonces, hemos tenido la introducción de [Elementos Personalizados](https://developer.mozilla.org/docs/Web/Web_Components/Using_custom_elements) y del [Manifesto Web Extensible](https://extensiblewebmanifesto.org/) alentando a que encontremos mejores formas de pavimentar los caminos de acceso al permitir a los desarrolladores explorar el espacio de los elementos y permitir a los organismos de estándares [actuar más como editores de diccionario](https://bkardell.com/blog/Dropping-The-F-Bomb-On-Standards.html).  A diferencia de los nombres de clase CSS, que podrían usarse para cualquier cosa, podemos estar mucho más seguros de que los autores que usaron un *elemento* no estándar realmente pretendían que fuera un elemento.  
```